src/preact/bento-ce.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO(alanorozco): These lint exceptions are required by initLogConstructor.
+/* eslint-disable import/no-restricted-paths */
+/* eslint-disable local/no-forbidden-terms */
+
+import {initLogConstructor} from '../log';
```

Comment:
I'm not sure how I should `initLogConstructor` safely.

Reply:
My instinct is that we shouldn't provide a logger beyond a stub that redirects to console